### DISCUSSION TITLE
Difference between gridpoints with and without /forecast

### DISCUSSION THREAD

**Discussion by martincmartin** [weather-gov/api#660] [General]
The url  https://api.weather.gov/gridpoints/BOX/66,92/forecast  is a forecast, but doesn't have quantity of precipitation.

The same URL without the /forecast suffix,  https://api.weather.gov/gridpoints/BOX/66,92 has `quantityOfPrecipitation`, hourly into the future.  So, it also seems to be a forecast, even though it doesn't have `forecast` in the URL.

What is the difference between these two URLs?  Why are there two different URLs that both seem to have forecast data?

**Comment by martincmartin** [comment]
I see the `points` API calls this `forecastGridData`.  What's the difference between `forecast` and `forecastGridData`, given that they both are about a single grid point?

**Reply by sullynole** [reply]
The Examples and Specification tabs here should help clarify the endpoints: https://www.weather.gov/documentation/services-web-api

**Comment by dwhitemv25** [comment]
A rough comparison is: 
* `/gridpoints` is the raw forecast data produced by the WFO, keyed by weather element
* `/gridpoints/.../forecast` is the processed version of `/gridpoints`, ready for publication on a forecast site, keyed by time

More information about the /gridpoints endpoint:

https://weather-gov.github.io/api/gridpoints
